src/apps/vpn/ui/screens/home/ViewServers.qml

```diff
@@ -132,7 +138,7 @@ Item {
             return;
         }
 
-        segmentedNav.stackContent.push(multiHopStackView);
+        segmentedNav.stackContent.push(multiHopLoader);
```

Comment:
Okay, so I know I am the one who implemented the segmented nav here, but can you think of any reason why we are manually pushing the multihop view/loader into the stacklayout inside of the segmented nav vs adding it via `stackContent` above?

`stackContent` is an alias for the StackLayout's `children` property which can take a list of views/items - I'm wonder if there was a reason as to why I chose to do this the "ugly" way, but if not, I'd like to do it the right way 🤔

Reply:
I think that was because we did have a time where multi-hop was desktop only. 
So in seg-nav you wrote `visible: stack.children.length > 1` - so if we did not add that multi-hop after complete, the whole thing was invisible.